src/contracts/Repository/Values/Content/URLWildcard/SearchResult.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Core\Repository\Values\Content\URLWildcard;
+
+use ArrayIterator;
+use eZ\Publish\API\Repository\Values\ValueObject;
+use Traversable;
+
+class SearchResult extends ValueObject implements \IteratorAggregate
```

Comment:
```suggestion
class SearchResult extends ValueObject implements IteratorAggregate
```

+ missing use \IteratorAggregate statement